run_localGPT.py

```diff
@@ -47,21 +47,11 @@ def load_model(device_type, model_id, model_basename=None):
 
     if model_basename is not None:
         if ".ggml" in model_basename:
-            logging.info("Using Llamacpp for GGML quantized models")
-            model_path = hf_hub_download(repo_id=model_id, filename=model_basename)
-            max_ctx_size = 2048
-            kwargs = {
-                "model_path": model_path,
-                "n_ctx": max_ctx_size,
-                "max_tokens": max_ctx_size,
-            }
-            if device_type.lower() == "mps":
-                kwargs["n_gpu_layers"] = 1000
-            if device_type.lower() == "cuda":
-                kwargs["n_gpu_layers"] = 1000
-                kwargs["n_batch"] = max_ctx_size
-            return LlamaCpp(**kwargs)
-
+            # Llamacpp no more supports GGML format - https://huggingface.co/TheBloke/Llama-2-13B-chat-GGML/discussions/14#:~:text=cpp%20is%20no%20longer%20compatible,As%20far%20as%20llama.
```

Comment:
I think the existing code can be improved but the new code is removing some existing functionality. Moreover, if you want to keep using GGML you can use an older version of `llama-cpp` locally. It's better for others to go ahead and start using new formats.

Reply:
OK thanks for the review..